types/frequency.ts

```diff
@@ -0,0 +1,51 @@
+// Last updated 2022-07-25
+export type ParquetModel = Array<ParquetColumn>;
+
+export type FrequencyParquetType = ParquetBaseType | ParquetStringType | ParquetNumericType | ParquetTemporalType;
+
+interface ParquetColumn {
+  name: string;
+  column_type: FrequencyParquetType;
+  compression: ColumnCompressionCodec;
+  bloom_filter: boolean;
+}
+
+type ColumnCompressionCodec = "UNCOMPRESSED" | "SNAPPY" | "GZIP" | "LZO" | "BROTLI" | "ZSTD" | "LZ4RAW";
+
+type ParquetBaseType = "BOOLEAN" | "INT32" | "INT64" | "FLOAT" | "DOUBLE" | "BYTE_ARRAY" | "FIXED_LENGTH_BYTE_ARRAY";
+
+type ParquetStringType = "STRING" | "UUID";
+
+type ParquetNumericType = ParquetInteger | ParquetDecimal;
+
+type ParquetInteger = {
+  INTEGER: {
+    bit_width: number;
+    sign: boolean;
+  };
+};
+
+type ParquetDecimal = {
+  DECIMAL: {
+    scale: number;
+    precision: number;
+  };
+};
+
+type ParquetTemporalType = "DATE" | "INTERVAL" | ParquetTime | ParquetTimestamp;
+
+type ParquetTime = {
+  TIME: {
+    is_adjusted_to_utc: boolean;
```

Comment:
if it's not adjusted to UTC should it have local Timezone? it seems complicated that's why maybe we need to only use UTC?

Reply:
UTC makes more sense and consumer can run their own adjustments to local time zome

Reply:
This is just from the Parquet spec, so Frequency supports it, so it is in the type here.